modules/cms/classes/Controller.php

```diff
@@ -488,11 +488,13 @@ protected function initTwigEnvironment()
 
         $useCache = !Config::get('cms.twigNoCache');
         $isDebugMode = Config::get('app.debug', false);
+        $strictVariables = (Config::get('cms.enableTwigStrictVariables', null) !== null) ?: $isDebugMode;
```

Comment:
Wouldn't this set $strictVariables = true when Config::get returns false?

Reply:
It's equivalent to:
```
if (Config::get('cms.enableTwigStrictVariables', null) !== null) {
    $strictVariables = Config::get('cms.enableTwigStrictVariables', null);
} else {
    $strictVariables = $isDebugMode;
}
```
or
```
$strictVariables = Config::get('cms.enableTwigStrictVariables', null);
if ($strictVariables !== null) {
    $strictVariables = $isDebugMode;
}
```
REF: https://en.wikipedia.org/wiki/Elvis_operator

Reply:
Today I need more coffee ... :coffee: